README.md

```diff
@@ -1 +1,69 @@
-# serverless-api
\ No newline at end of file
+# AWS API-Gateway, Dynamo and Lambda
+
+## Author: Eva Grace Smith
+
+### Task
+
+Create a serverless REST API.
+
+### Requirements
+
+Create a single resource REST API using a domain model, constructed using AWS Cloud Services.
+
+### Database: DynamoDB
+1 Table required.
+
+### Routing: API Gateway
+
+
+### UML
+
+![UML](./img/lab18-UML.png)
```

Comment:
We aren't connecting to DynamoDB from API Gateway, but from our Lambda function.